packages/api-v4/src/cloudpulse/types.ts

```diff
@@ -116,3 +116,89 @@ export interface CloudPulseMetricsList {
   metric: { [resourceName: string]: string };
   values: [number, string][];
 }
+
+export interface CreateAlertDefinitionPayload {
+  name: string | null;
+  region: string | null;
+  description?: string | null;
+  service_type: string | null;
+  engineOption: string | null;
+  resource_ids: string[];
+  severity: string;
+  criteria: MetricCriteria[];
+  triggerCondition: {
+    criteria_condition: string;
+    polling_interval_seconds: string;
+    evaluation_period_seconds: string;
+    trigger_occurrences: number;
+  };
+  notifications: {
+    notification_type: string;
+    content: {
+      email: string;
+    };
+  }[];
+  sink_ids: string[];
+}
```

Comment:
```suggestion
  triggerCondition: TriggerCondition;
  notifications: Notification[];
  sink_ids: string[];
}

interface Notification {
    notification_type: string;
    content: {
      email: string;
    };
  }
  
  interface TriggerCondition {
    criteria_condition: string;
    polling_interval_seconds: string;
    evaluation_period_seconds: string;
    trigger_occurrences: number;
  }
```

name the interfaces accordingly